README.md

```diff
@@ -2,28 +2,36 @@
 
 UI for events from [Glutton](https://github.com/mushorg/glutton), streamed live from a development instance.
 
-## Development Requirements
-For backend development
+## Development Requirements -
```

Comment:
Why adding the dash?

Reply:
Added just to differentiate from heading under it. Will remove it now.